docs/tasks/run-application/update-api-object-in-place.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Update API Objects in Place Using kubectl patch
```

Comment:
I want to keep this under Tasks, not Concepts. Yes, I think we should create other Update tasks and consolidate them under a heading. For now, let's leave it where it is.

I changed the filename to update-api-object-kubectl-patch.md